plugins/woo-ai/package.json

```diff
@@ -26,7 +27,31 @@
 		"prettier": "npm:wp-prettier@^2.8.5",
 		"ts-loader": "^9.5.1",
 		"typescript": "^5.3.3",
-		"uglify-js": "^3.17.4"
+		"uglify-js": "^3.17.4",
+		"wireit": "^0.14.1"
```

Comment:
In a few packages you've included wireit when not using it for any scripts yet. Just checking if thats intentional or needed?

Reply:
I used an automated script that added it to all of our projects. With a few small exceptions though, sooner rather than later, all of them should be using `wireit`.